doc/src/learn/architecture/consensus.md

```diff
@@ -1,20 +1,23 @@
 ---
-title: Narwhal and Tusk, Sui's Consensus Engine
+title: Narwhal, Bullshark, and Tusk, Sui's Consensus Engine
 ---
 
-This is a brief introduction to [Narwhal and Bullshark](https://github.com/MystenLabs/narwhal), the high-throughput mempool and consensus offered by Mysten Labs. Sui runs consensus as needed to periodically checkpoint its state. And for those transactions that require a total ordering, Narwhal/Bullshark is the consensus engine of Sui.
+This is a brief introduction to [Narwhal, Tusk](https://github.com/MystenLabs/narwhal), and [Bullshark](https://arxiv.org/abs/2201.05677), the high-throughput mempool and consensus offered by Mysten Labs. Sui runs consensus as needed to periodically checkpoint its state. And for those transactions that require a total ordering, Narwhal and either Bullshark or Tusk comprise the Sui Consensus Engine.
 
 The dual name highlights that the systems split the responsibilities of:
-- ensuring the availability of data submitted to consensus ([Narwhal](https://arxiv.org/abs/2105.11827))
-- agreeing on a specific ordering of this data ([Bullshark](https://arxiv.org/abs/2201.05677))
+- ensuring the availability of data submitted to consensus = [Narwhal](https://arxiv.org/abs/2105.11827)
+- agreeing on a specific ordering of this data = [Bullshark](https://arxiv.org/abs/2201.05677) or [Tusk ]
```

Comment:
extra space and no URL to form the link
[Tusk ]

Reply:
Fixed.